# S2Solution.msg

std_msgs/Header header
int32 author_vin
int32 winner_vin



